cpp/src/arrow/adapters/arrow-rados-cls/cls_arrow.cc

```diff
@@ -126,6 +155,15 @@ class RandomAccessObject : public arrow::io::RandomAccessFile {
   std::vector<ceph::bufferlist*> chunks_;
 };
 
+/// \brief Scan the Parquet file objects.
```

Comment:
Scan RADOS objects containing Parquet binary data. It is always better to make the descriptions a little more descriptive.

Reply:
Done. Didn't know the exact specifics. Fixed now.